utils/templateDockerFile

```diff
@@ -21,7 +21,17 @@ RUN apt-get update && \
     ssh \
     tar \
     gzip \
-    ca-certificates
+    ca-certificates \
+    julia
```

Comment:
Do we absolutely have to ? I don't mind but it's adding a lot of stuff for an edge case use